ruby/vampires.rb

```diff
@@ -0,0 +1,70 @@
+
+applicant_name = #{name}
+  puts "what is your name"
+  name = gets.chomp
+
+def age_correct
+  puts "what is your age"
+  age = gets.chomp.to_i
+  puts "what year were you born"
+  birth_year = gets.chomp.to_i
+if 2017-birth_year == age
+  age_correct = TRUE
```

Comment:
`TRUE` is actually a different data type than a boolean.  All caps makes this what we call a constant.

Change your TRUE and FALSE to be all downcase to use the boolean format.